pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1646,20 +2003,21 @@ func TestSingleConstraint(t *testing.T) {
 
 func TestMultipleConstraints(t *testing.T) {
```

Comment:
We should have a test for the case where different constraints have different policies.

Reply:
yes, still in progress now.